superset/models/helpers.py

```diff
@@ -1340,14 +1340,19 @@ def get_time_filter(  # pylint: disable=too-many-arguments
             )
         return and_(*l)
 
-    def values_for_column(self, column_name: str, limit: int = 10000) -> list[Any]:
-        # always denormalize column name before querying for values
+    def values_for_column(
```

Comment:
Do our unit tests already have cases where it covers if `denormalize_column` returns the correct values for both true and false already? If not, I would add tests to confirm that the correct things are returned for when the flag is true since we're defaulting to false.

Reply:
@sadpandajoe @villebro I just added some basic tests to https://github.com/apache/superset/pull/26220 (since this one got merged). It doesn't test the logic implemented in the DB engine level to denormalize a column (I believe this is DB-specific and would require a more complex setup), but should be at least testing the business logic.